lib/service/cache/redis.js

```diff
@@ -51,11 +51,22 @@ class Redis extends Service {
    * @returns {Promise}
    */
   _initSequence() {
+    // @deprecated @todo Remove those lines for Kuzzle v3 then 
```

Comment:
```suggestion
    // @deprecated Remove those lines for Kuzzle v3 then 
```
We usually only add one of those tags, in this case it would be` @deprecated`